lib/buckets.ps1

```diff
@@ -39,12 +39,17 @@ function apps_in_bucket($dir) {
     return Get-ChildItem $dir | Where-Object { $_.Name.endswith('.json') } | ForEach-Object { $_.Name -replace '.json$', '' }
 }
 
+function Get-LocalBucket {
+    <#
+    .SYNOPSIS
+        List all local buckets.
+    #>
+
+    return (Get-ChildItem $bucketsdir).Name
+}
+
 function buckets {
-    $buckets = @()
-    if(test-path $bucketsdir) {
-        Get-ChildItem $bucketsdir | ForEach-Object { $buckets += $_.Name }
-    }
-    return $buckets
+    return Get-LocalBucket
```

Comment:
Would it be a good idea to add a deprecated warning by using something like this?

```pwsh
function Show-DeprecatedWarning($invocation, $new) {
    Write-Host ('"{0}" will be deprecated. Please change your code/manifest to use "{1}"' -f $invocation.MyCommand.Name,$new) -f DarkYellow -NoNewline
    Write-Host " -> $($invocation.PSCommandPath):$($invocation.ScriptLineNumber):$($invocation.OffsetInLine)" -f DarkGray
}


function buckets {
	Show-DeprecatedWarning $MyInvocation 'Get-LocalBucket'
    return Get-LocalBucket
}
```

Just for functions that _might_ get used in manifests.

Reply:
![image](https://user-images.githubusercontent.com/13260377/56468532-357a6e00-642e-11e9-9a36-e028a85d618b.png)

Done. Added it to core

Reply:
This will be nice start to proper changelogs after bucket folder extraction. There could be then normal github release of scoop with some changelog list, which could contain few parts (added, changed, deprecated)